testing/simapp/app.go

```diff
@@ -122,8 +122,10 @@ import (
 	ibcmock "github.com/cosmos/ibc-go/v7/testing/mock"
 	simappparams "github.com/cosmos/ibc-go/v7/testing/simapp/params"
 	simappupgrades "github.com/cosmos/ibc-go/v7/testing/simapp/upgrades"
+	v5 "github.com/cosmos/ibc-go/v7/testing/simapp/upgrades/v5"
 	v6 "github.com/cosmos/ibc-go/v7/testing/simapp/upgrades/v6"
 	v7 "github.com/cosmos/ibc-go/v7/testing/simapp/upgrades/v7"
+	v71 "github.com/cosmos/ibc-go/v7/testing/simapp/upgrades/v7.1"
```

Comment:
Happy to change this upgrade package structure, open to suggestions if anyone thinks a directory/pkg per upgrade is overkill.

Reply:
We could maintain all under `package upgrades` and use for example: `UpgradeNameV7`, `UpgradeNameV6`..etc or something

Reply:
I think my personal preference would be for all the names to be separate variables within one package, but it's not really a big deal either way.

Reply:
Any thoughts @colin-axner?

Reply:
Sounds good to me! My only nit is using something like `UpgradeNameV7_1` or `UpgradeNameV7dot1` or something instead of `UpgradeNameV71`